### ISSUE TITLE
If no classes in current timeframe, display first week of classes.

### ISSUE THREAD

**Issue by MikeiLL** [MikeiLL/mz-mindbody-api#15]
This should be the case with events as well, but there's an easy workaround for that which is just a longer Option period for Events display cycle.

Or could display empty calendar starting on current week.

This would require either a second call to the API or the initial call would need to have an arbitrary end date. A second call to the API seems to make sense, then display first week of classes within the

**Comment by MikeiLL** [comment]
new branch https://github.com/MikeiLL/mz-mindbody-api/tree/noclasses

**Comment by rtzee** [comment]
I think this is really only important for a new studio that is launching before classes actually start. Of course they can link to MBO schedule in the interim. Could we *only* trigger the second API call if the first call returned an empty set? That seems like the best strategy to reduce the overhead.

**Comment by MikeiLL** [comment]
Yes. I think that makes sense. I'm actually not quite sure how to test this. Not sure if I can remove ALL the classes for current week from the Sandbox MBO site (`-99` account).

**Comment by rtzee** [comment]
Yep. we won't have classes until next week so we get an empty result when calling the current week. This is what lead me to change the "error" message in inc/frontend/views/horizontal_schedule.php

**Comment by MikeiLL** [comment]
Actually it may by `$this->schedule_object->classes['GetClassesResult']['ResultCount']` on line #234 that we want to use in `inc/schedule/class-display.php`.

**Comment by rtzee** [comment]
So would we have to keep making sequential API requests until we got a non zero result for ResultCount? Or is there an API call to get the "next available class"  and then use that date to determine the date range of the corresponding week?

**Comment by MikeiLL** [comment]
> Or is there an API call to get the "next available class" and then use that date to determine the date range of the corresponding week?

Good question. Doesn't look like it https://developers.mindbodyonline.com/PublicDocumentation/ClassService?version=v5.1

**Comment by rtzee** [comment]
So I guess we'd need to do a larger range like 60 days on the second API call and then, if there was a class, a third API call to show the correct week. 

Kinda sounds like a lot of work for a really small use case.

**Comment by MikeiLL** [comment]
The code I just pushed out might achieve this. Would you mind seeing what happens if you use it without current classes?

**Comment by MikeiLL** [comment]
(on the `noclasses` branch.)
